Project.toml

```diff
@@ -1,5 +1,5 @@
 name = "Pkg"
-uuid = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+uuid = "44cfe95a-1eb2-52ea-b672-e2afdf69b780"
```

Comment:
This should be reverted.

Reply:
Hah. Funny that our CI doesn't catch that.

Reply:
Woow.